test.env

```diff
@@ -0,0 +1,2 @@
+TORII_URL=127.0.0.1:1337
```

Comment:
I don't think it is a good idea to keep test config in the root, if it is only needed for the test you can move it to the same folder with that test

Reply:
Moved into `./tests`.